diffsims/libraries/structure_library.py

```diff
@@ -56,6 +56,36 @@ def __init__(self,
         for ident, struct, ori in zip(identifiers, structures, orientations):
             self.struct_lib[ident] = (struct, ori)
 
+    def get_library_size(cls, to_print = False):
```

Comment:
This is a "method" rather than a "class method". It's probably more jargon than you need, but you should have `self `rather than `cls`

Reply:
Thank you! I was unsure about this, I am used to C++, so some of the Python-class stuff is new to me :) Fixed in commit norskie/norskie:get_struclib_size@#77